.pipelines/templates/aks-setup.yaml

```diff
@@ -8,12 +13,22 @@ steps:
 
       az group create -n ${AZURE_CLUSTER_NAME} -l ${AKS_CLUSTER_LOCATION} > /dev/null 2>&1
 
-      az aks create \
-          -g ${AZURE_CLUSTER_NAME} \
-          -n ${AZURE_CLUSTER_NAME} \
-          --enable-managed-identity \
-          --no-ssh-key \
-          > /dev/null 2>&1
+      if [[ "${{ parameters.aksAddonRun }}" == "True" ]]; then
+        az aks create \
+            -g ${AZURE_CLUSTER_NAME} \
+            -n ${AZURE_CLUSTER_NAME} \
+            --enable-managed-identity \
+            --no-ssh-key \
+            --kubernetes-version ${K8S_VERSION} \
```

Comment:
where is this defined?

Reply:
K8S_VERSION is set as an UI configurable variable here: https://dev.azure.com/AzureContainerUpstream/OpenServiceMesh/_build?definitionId=330&_a=summary. Edit -> variables